requirements.txt

```diff
@@ -1,4 +1,5 @@
 setuptools
 lxml>=4.3.0
 numpy>=1.12.0
-webrtcvad-wheels>=2.0.10
\ No newline at end of file
+webrtcvad-wheels>=2.0.10
+Pillow==9.4.0
```

Comment:
Any reason to have this specific version? Could it be `Pillow>=9.0.0` ?

Reply:
I have changed the version specification. Pillow is just used for an image convert🤔